src/components/navbar-header-panels/dropdown-menu-panel.component.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { render } from "react-dom";
+import "carbon-components/css/carbon-components.min.css";
+import { Dropdown } from "carbon-components-react";
```

Comment:
Shouldn't import from root module; instead make an import from the respective submodule.

Reply:
Thanks for the review
 @FlorianRappl